repos/c2sm/packages/py-icon4py/package.py

```diff
@@ -66,7 +67,8 @@ def test(self):
 
         # unit tests
         if 'py-pytest-mpi' in self.spec:
-            python('-m', 'pytest', '--with-mpi', '-v', '-s')
+            python('-m', 'pytest', '--with-mpi', '-v', '-s', '-m',
```

Comment:
What happens for older version where `not slow_tests` is not present?

Reply:
Pytest will still execute all tests. If you run pytest with the command pytest -m "not slow_tests" and the "slow_tests" marker is not found anywhere in the test files, pytest will still execute all tests, regardless of whether they have any markers or not.